src/cache-restore.ts

```diff
@@ -0,0 +1,36 @@
+import * as cache from "@actions/cache";
+import * as core from "@actions/core";
+
```

Comment:
```suggestion
```

I think there's some Eslint-rules here that should prevent empty lines between imports.